Reqnroll/CucumberMessages/Configuration/ICucumberMessagesConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Reqnroll.CucumberMessages.Configuration
+{
+    public interface ICucumberMessagesConfiguration
+    {
+        bool Enabled { get; }
+        string BaseDirectory { get; }
```

Comment:
I was trying to trace down all the code paths, but as far as I have seen the `BaseDirectory`, the `OutputDirectory` and the `OutputFileName` are all in all just used to build the output file path.
My suggestion would be to merge all these to an optional `OutputFilePath` that defaults to "./reqnroll_report.ndjson" if it is not specified. Once we find a good use-case for separation (e.g. we will generate multiple files) we can still introduce some base-folder concept.

Reply:
See this [commit](https://github.com/reqnroll/Reqnroll/pull/233/commits/36dc50b19759e50e23366c27cf1f08cbcfe6928b)
The three parts of the path are consolidated into a single configuration element. This is reflected in both the configuration file and in the Environment variable overrides.